chainer/links/connection/gru.py

```diff
@@ -77,6 +79,30 @@ class StatelessGRU(GRUBase):
 
     .. seealso:: :class:`~chainer.links.StatefulGRU`
 
+    .. admonition:: Example
+
+        There are several ways to make a StatefulGRU link.
```

Comment:
This should be StatelessGRU, I guess.
Also please put ``` ``  `` ``` to make it a code fragment.